### ISSUE TITLE
Applying RE task to n2c2 dataset TrAP task, results are highly biased towards label 0 

### ISSUE THREAD

**Issue by v-loves-avocados** [dmis-lab/biobert#79]
I tested BioBERT for relation extraction. Unfortunately, the result didn't seem right.

The task is Treatment is administered for medical problem (TrAP) .

Example: He was given **Lasix** periodically to prevent him from going into **congestive heart failure**.

Dataset: n2c2 2010 relation dataset.   
./concept_assertion_relation_training_data/beth/rel

Sample processed training data:
![image](https://user-images.githubusercontent.com/52256615/72699944-2fb9d200-3aff-11ea-880c-9443576f327a.png)

Sample processed testing data:
![image](https://user-images.githubusercontent.com/52256615/72699973-452efc00-3aff-11ea-9fab-e01749587838.png)

The command is basically the exact same replication of the code on the github. 

RE_DIR = './eval_trap'
TASKNAME = 'gad' 
BIOBERT_DIR = './biobert_v1.1_pubmed'

!python ./biobert/run_re.py \
    --task_name={TASKNAME} \
    --do_train=true \
    --do_eval=true \
    --do_predict=true \
    --vocab_file=$BIOBERT_DIR/vocab.txt \
    --bert_config_file=$BIOBERT_DIR/bert_config.json \
    --init_checkpoint=$BIOBERT_DIR/model.ckpt-1000000 \
    --max_seq_length=128 \
    --train_batch_size=32 \
    --learning_rate=2e-5 \
    --num_train_epochs=3.0 \
    --do_lower_case=false \
    --data_dir=$RE_DIR/ \
    --output_dir=./eval_trap/trap_output2/

In both the training and testing datasets, the 0 and 1 labels are 50-50, balanced.

-------------------
The results seem wrong, but in an interesting way.
There are only two types of results:

{0.81744546, 0.18255456} and {0.81289905, 0.18710092}. 

The interesting thing is that, the ones with {0.81289905, 0.18710092 (slightly bigger towards label '1')} all correspond to the label '1', all of them. And the ones with {0.81744546, 0.18255456} all correspond to label '0'.  

I am wondering that as you read through this email. do you see any obvious problems? Thank you!

**Comment by v-loves-avocados** [comment]
Problem solved - I pointed the re.py to a wrong data directory. Sorry!
